hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OzoneManager.java

```diff
@@ -3125,6 +3203,8 @@ public S3VolumeContext getS3VolumeContext() throws IOException {
           multiTenantManager.getTenantForAccessID(accessId);
 
       if (optionalTenantId.isPresent()) {
+        // To block all S3 multi-tenancy requests if disabled
```

Comment:
Let's say initially multi-tenancy was enabled, and let's say we created the tenant volume and some buckets and keys. Now someone disables multi-tenancy. This point in code will take care of requests that will come through S3. What happens to access to multi-tenant volumes through ozone sh or OFS ?

Reply:
The offline answer to this question was that access to multi-tenant data through non-S3 methods would still work as long as Ranger was still configured with acls to allow access.

Reply:
Yes. Non-S3 requests like `ozone sh` or HCFS (ofs/o3fs) ones should not even go through `getS3VolumeContext()` call.